docs/zkapps/advanced-snarkyjs/time-locked-accounts.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Time-Locked Accounts
+hide_title: true
+---
+
+:::info
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+# Time-Locked Accounts
+
+Time-locking allows you to pay someone in MINA or custom other tokens subject to a vesting schedule. This means that the tokens are initially locked, and only become available for withdrawal after a certain time, or gradually according to a certain schedule.
+
+The zkApp feature that enables time-locking is the `timing` field, which is present on every account. It look like this:
+
+```ts
+type Account = {
+  // ...
+  timing: {
+    isTimed: Bool;
+    initialMinimumBalance: UInt64;
+    cliffTime: UInt32;
+    cliffAmount: UInt64;
+    vestingPeriod: UInt32;
+    vestingIncrement: UInt64;
+  };
+};
+```
+
+`isTimed` indicates whether this account is time-locked. The other fields are parameters that allow us to define a vesting schedule in a very flexible manner. We will explain them in detail below. By default, accounts are not time-locked, and `isTimed` will be `false` and all other properties will contain default values.
```

Comment:
`time-locked` >  `time locked`
When it's a word doesn't follow it, like accounts.

2 places in this paragraph. I missed it earlier.